xorg-x11-drv-nvidia.spec

```diff
@@ -468,7 +492,7 @@ fi ||:
 %files kmodsrc
 %dir %{_datadir}/nvidia-kmod-%{version}
 %{rpmmacrodir}/macros.%{name}-kmodsrc
-%{_datadir}/nvidia-kmod-%{version}/nvidia-kmod-%{version}-x86_64.tar.xz
+%{_datadir}/nvidia-kmod-%{version}/nvidia-kmod-%{version}-*.tar.xz
```

Comment:
Same simplification,but wildcard is fine also.

Reply:
switched to %{_arch} also